### PR TITLE
feat(landingpage): finish about page

### LINKED ISSUE #810: landingpage: Finish about page
- Briefly describe the history of the invoex-elements (see README, Docs, workshop slides, ...) in `about#history`
- Embed the Github activity graph of our repository if possible (`about#activity`)
- Contributors @ inovex elements: add roles to each member and add designers (`about#team`)

### PR DESCRIPTION
**Pull request by ninaschlz**
Closes #810 

## Proposed Changes

- Contributors: Add designers to list and a role to each contributor
- History: Describe history of inovex elements
- Activity: Graph with repo contributions the last 12 months

### CODE REVIEW

File: packages/landingpage-new/components/about/activity.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Filler,
+  Legend
+);
+
+const months: string[] = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```
Comment (janivo) [inline packages/landingpage-new/components/about/activity.tsx:27]: Can we used the short names here to reduce noise? `Jan 22`, `Feb 22`, ...

**Review by janivo — CHANGES_REQUESTED** [review]
Great job overall. Looks very nice! Also nice, that you found a way to include our designers 👍🏼 
There are just some technical aspects I would like to change

File: packages/landingpage-new/components/about/activity.tsx [hunk 2]
```diff
@@ -24,96 +25,116 @@ ChartJS.register(
...
       datasets: [
         {
           fill: true,
           label: 'contributions',
-          data: graphActivities.map(graphActivity => graphActivity.contributions),
+          data: contributions,
           borderColor: '#4F46FF',
           backgroundColor: '#BECCFF50',
         },
       ],
```
Comment (janivo) [inline packages/landingpage-new/components/about/activity.tsx:124]: Can we move this to the `useEffect` hook above? That way, the `label` and `contributions`  will be unnecessary and all of our data lies in `graphData`

**Review by janivo — CHANGES_REQUESTED** [review]
When navigating away and to the about page, this error appears:
image

**Review by janivo — CHANGES_REQUESTED** [review]
image

The activity graph appears to be empty when running `yarn build` and `yarn start` 🤔

**Comment by ninaschlz** [comment]
> image
> 
> The activity graph appears to be empty when running `yarn build` and `yarn start` 🤔

Strange... I can't reproduce this. In my case all data are present

**Review by janivo — APPROVED** [review]

**Review by janivo — APPROVED** [review]

**Pull request merged**